docs/actions/bazarliste.yaml

```
@@ -249,33 +274,47 @@ actions:
         label: Hauteur de la carte
         type: text
         default: 600px
+      entrydisplay:
+        label: Lors du click, afficher la fiche
```

```suggestion
        label: _t(AB_bazarliste_entry_display_label)
```
pourrait dès maintenant introduire les variables de traductions (nom à déterminer en fonction du besoin).
Ce serait plus agréable pour le chantier traduction que toute nouvelle entrée dans le fichier bazarliste.yaml soit déjà dans le fichier de traduction (au moins en Français)